Client.xcodeproj/project.pbxproj

```diff
@@ -19305,6 +19060,11 @@
 			package = 43C6A47D27A0679300C79856 /* XCRemoteSwiftPackageReference "MappaMundi" */;
 			productName = MappaMundi;
 		};
+		F8DEACC42A3D43DA00C3B19D /* Sentry */ = {
+			isa = XCSwiftPackageProductDependency;
+			package = 5A37861729A2C337006B3A34 /* XCRemoteSwiftPackageReference "sentry-cocoa" */;
+			productName = Sentry;
+		};
 /* End XCSwiftPackageProductDependency section */
```

Comment:
Strangely sentry got removed and re-added. Shouldn't cause any issue though

Reply:
So it looked like Sentry was being accessed via the `SyncTelemetry` target for the `Accounts` target IIRC. Because when I removed the `SyncTelemetry` module suddenly the `Accounts` module was complaining about not having a Sentry reference. 🤷🏾‍♀️